### PR TITLE
Add toolbar and headline section

### PR DESCRIPTION
**Pull request by ivangonzalez224**
It is my personal developer portfolio and I added the followings new features:

- Add tool bar and headline section html skeleton.
- Add the logo and menu button.
- Add header and social media buttons.
- Add the illustration image.
- Add styles for tool bar and headline sections.
- Update the README.md file with a relevant description.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                <ul class="navMenu">
+                    <li>
+                        <a class="navBtnMenu" href="#">
+                            <img class="menuBtn" src="images/menuUnion.png" alt="">
+                        </a>
+                    </li>
+                </ul>
+            </nav>   
+        </div>
+    </div>
```
Comment (Tufahel) [inline index.html]: - [ ] This section should be inside the `header` tag. Kindly fix this issue.
- Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                        
+                    </div>
+                </div>
+
+                <div class="headlineImage">
+                    <img src="images/illustration.png" alt="">
+                </div>
+            </header>
+        </div>
+    </div>
```
Comment (Tufahel) [inline index.html]: - [ ] This section should be inside `section` tag. Kindly replace `div` tag with `section` tag, as it represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                        </a>
+                    </li>
+                </ul>
+            </nav>   
+        </div>
+    </div>
+
+    <div class="headline-container">
+        <div class="wrapper">
+            <header>
```
Comment (Tufahel) [inline index.html]: - [ ] Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. So there should not any `header` tag. Kindly remove this.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                    </div>
+                </div>
+
+                <div class="headlineImage">
+                    <img src="images/illustration.png" alt="">
+                </div>
+            </header>
+        </div>
+    </div>
+</body>
```
Comment (Tufahel) [inline index.html]: - No images are working in your project. 
- [ ] `First:`  There should be a menu-hamburger icon.
- [ ] `Second:` There should be a `background image`. Kindly use `background-image` property of  CSS for this section.
- [ ]  `Third:` There should be `social media` icons with links mentioned  in the template.
- image

- I can see you have tried to use images using `img` tag ``, but there is no `images` folder in your repository. May be that is the issue the images and icons are not working. Kindly fix this issue.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+  justify-content: space-between;
+  height: 48px;
+}
+
+nav ul {
+  display: flex;
+}
+
+nav .logo {
+  font-family: $'Roboto';
```
Comment (Tufahel) [inline style.css]: - [ ] The usage of fonts are not correct in your project. Kindly fix every font. Here is a demo [Link](https://www.loom.com/share/9912309eebdb46b698a3243a1c92e2c3) how to correctly use a font.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @ivangonzalez224 :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Linter checks are passing.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bienvenuushindi — APPROVED** [review]
# Approved 👏  

## Hi @ivangonzalez224    👋, 

Well done implementing the requested changes #3 👏

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

### Suggestion

- NOTE: **It's not good  to open a new PR when requesting for a re-review. Always use initial PR for all re-reviews. Please  Keep the same PR next time until it get approved.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @bienvenuushindi  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**